ricaun-io/ricaun.RevitTest
# Version 1.3.5

### Features
- Update `Application` images to support `light` and `dark` theme.
### Application
- Update project to build `net47` and `net48`.
- Update `Images` with light and dark theme.
- Update `ricaun.Revit.UI` with the image auto theme based in the `light` and `dark` in the file name.
- Update `Exception` when null.
- Update timestamp in temp folder to hexadecimal.
- Update test folder to temporarty the folder `RevitTest` to reduce the `PathTooLongException`. (Fix #48)
- Update to use `ricaun.Revit.UI.Tasks` library to run async tests.
- Update to use `ricaun.Revit.UI.Busy` library to check if revit is busy.
### Console
- Throw exception when `RevitTestUtils` is empty.
### TestAdapter
- Show Warning when Console fails.
- Fix local file copy to temporary by zip folder.
- Add `TestsFilePathTooLong` to similate long path inside test.
- Update to use `ricaun.Revit.UI.Tasks` library to run async tests.
- Update `ApplicationUtils` check local file exists.
- Update `RevitTestConsole` with `SetCurrentDirectory` using the source test file.
